proto/buf/alpha/audit/v1alpha1/event.proto

```diff
@@ -73,5 +74,15 @@ message Event {
   // Time of the audited event. It specifies "when" it happened.
   google.protobuf.Timestamp event_time = 5;
   // Payload of the audited event. It specifies additional context on the event.
-  // TODO add oneof payload events.
+  oneof payload {
+    PayloadOrganizationCreated organization_created = 6;
+  }
+}
+
+// PayloadOrganizationCreated is the payload for an ORGANIZATION_CREATED event.
+message PayloadOrganizationCreated {
+  // name is the name of the created organization.
+  string name = 1;
```

Comment:
Oh sorry, [I meant adding it back to the `actor` and `resource` messages](https://github.com/bufbuild/core/pull/5867#discussion_r961678823), not to each specific payloads. 😅

Reply:
ah, yeah, I can add it there. sorry, not enough coffee yet 😄

Reply:
Done in https://github.com/bufbuild/buf/pull/1375/commits/faa557bd7bcd70360f3a0c6e16aa678896240126!